### DISCUSSION TITLE
I May Misunderstand the purpose of this app?

### DISCUSSION THREAD

**Discussion by richardstevenhack** [paulrobello/parllama#6] [Q&A] [answered]
The description says this is for the "use and maintenance" of LLMs.

I have explored all the options. It appears that the purpose of this app is not to use LLMs from the command line or this app, i.e., engage in chat with the LLM or otherwise interact with it, but instead to enable manipulation of the LLM for the purpose of creating a variant LLM for submission to Ollama.

Do I have this correct? I thought it was a way to communicate with the installed LLM.

**Comment by paulrobello** [comment, marked as answer]
The initial release was purely to help maintain / create / augment your models.
I am currently working on the portion that will let you interact / chat with the LLM. I hope to have a rough implementation of that in by end of week.
The project is still in heavy flux and suggestions / requests are more than welcome.
I will add a roadmap to the readme to help with any confusion of where its at / going.

**Comment by richardstevenhack** [comment]
OK, got it. The main advantage I see with the current implementation is the ability to both pull and see the list of available Ollama models without having to leave the interface and use a browser to get the list. Some of the other TUIs don't have that. I mostly use GUIs myself but I'm keeping an eye on the various TUIs as well for testing.

**Reply by paulrobello** [reply]
I added a chat tab to v0.2.4. Still very work in progress.

**Reply by richardstevenhack** [reply]
Just tested it. Works fine. Good work.
